soh/soh/Enhancements/randomizer/3drando/item.hpp

```diff
@@ -127,7 +127,7 @@ class Item {
         return IsAdvancement();
     }
 
-    const uint32_t GetHintKey() const {
+    uint32_t GetHintKey() const {
```

Comment:
In case anyone else didn't know because I had to look it up, `const` in front of a function declaration means it's returning a `const` reference. Since we aren't actually returning a reference here, I think the removed `const` keyword was doing basically nothing.